src/SWP/Bundle/ContentBundle/Model/ArticleMedia.php

```diff
@@ -0,0 +1,272 @@
+<?php
+
+/**
+ * This file is part of the Superdesk Web Publisher Content Bundle.
+ *
+ * Copyright 2016 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2016 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+namespace SWP\Bundle\ContentBundle\Model;
+
+use SWP\Component\Bridge\Model\ItemInterface;
+
+/**
+ * ArticleMedia represents media which belongs to Article.
+ */
+class ArticleMedia implements ArticleMediaInterface
+{
+    /**
+     * @var string
+     */
+    protected $id;
+
+    /**
+     * @var FileInterface
+     */
+    protected $file;
+
+    /**
+     * @var ImageInterface
+     */
+    protected $image;
+
+    /**
+     * @var ArticleInterface
+     */
+    protected $article;
+
+    /**
+     * @var string
+     */
+    protected $description;
+
+    /**
+     * @var string
+     */
+    protected $located;
+
+    /**
+     * @var string
+     */
+    protected $byLine;
+
+    /**
+     * @var string
+     */
+    protected $body;
+
+    /**
+     * @var string
+     */
+    protected $mimetype;
+
+    /**
+     * @var string
+     */
+    protected $usageTerms;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * @return FileInterface
+     */
```

Comment:
`{@inheritdoc}` should be used for all getters and setters here (see an interface)